app/controllers/users_controller.rb

```diff
@@ -1,40 +1,140 @@
 # frozen_string_literal: true
```

Comment:
FYI
It might be appropriate to deal with some of the following in this PR.
(I'm too tired to dig into them tonight.)
```sh
$ rubocop --ignore-parent-exclusion app/controllers/users_controller.rb -A
Inspecting 1 file
C

Offenses:

app/controllers/users_controller.rb:48:3: C: Metrics/AbcSize: Assignment Branch Condition size for show is too high. [<9, 17, 7> 20.47/20]
  def show ...
  ^^^^^^^^
app/controllers/users_controller.rb:84:5: C: Style/GuardClause: Use a guard clause (return unless @user2.bonuses) instead of wrapping the code inside a conditional expression.
    if @user2.bonuses
    ^^
app/controllers/users_controller.rb:86:17: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over unannotated tokens (like %s).
        format("%-6d %s", points, reason.gsub(/\s+/, " "))
                ^^^^
app/controllers/users_controller.rb:86:22: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over unannotated tokens (like %s).
        format("%-6d %s", points, reason.gsub(/\s+/, " "))
                     ^^
app/controllers/users_controller.rb:92:3: C: Metrics/AbcSize: Assignment Branch Condition size for update is too high. [<16, 30, 9> 35.17/20]
  def update ...
  ^^^^^^^^^^
app/controllers/users_controller.rb:112:5: C: Style/GuardClause: Use a guard clause (return if errors) instead of wrapping the code inside a conditional expression.
    unless errors
    ^^^^^^
app/controllers/users_controller.rb:144:3: C: Metrics/AbcSize: Assignment Branch Condition size for show_selected_users is too high. [<4, 23, 8> 24.68/20]
  def show_selected_users(query, args = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/users_controller.rb:144:3: C: Metrics/MethodLength: Method has too many lines. [32/30]
  def show_selected_users(query, args = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```